code/drasil-example/Drasil/DblPendulum/Concepts.hs

```diff
@@ -9,10 +9,27 @@ pendulum :: CI
 pendulum = commonIdeaWithDict "pendulum" (pn "Pendulum") "Pendulum" [physics]
 
 concepts :: [IdeaDict]
-concepts = [nw rod] 
+concepts = map nw [rod, horizontal, vertical] ++ map nw defs 
        
-rod :: NamedChunk
+rod, horizontal, vertical :: NamedChunk
 rod = nc "rod" (cn' "rod")
+horizontal = nc "horizontal" (cn "horizontal") 
+vertical = nc "vertical" (cn "vertical") 
+---
+
+defs:: [ConceptChunk]
+defs = [shm, amplitude, component, arcLen, motion]
+
+shm, amplitude, component, arcLen, motion :: ConceptChunk
+shm = dcc "SHM" (nounPhraseSP "simple harmonic motion") ("repetitive movement back and forth through" ++
+                                                         "an equilibrium position such that the maximum" ++
+                                                         "displacement on one side is equal to the maximum" ++
+                                                         "displacement on the other side")
+amplitude = dcc "amplitude" (nounPhraseSP "amplitude") ("the maximum displacement of a body measured from" ++
```

Comment:
We might want to use amplitude for something other than displacements.  We talk about amplitude for any sine wave, whether the dependent variable is displacement, or voltage, or current, or whatever.  The Wikipedia page for the  Sine wave (https://en.wikipedia.org/wiki/Sine_wave) has a definition that I like:

amplitude: the peak deviation for the function from zero

Reply:
I'm tempted to classify the definition of amplitude as belonging to `Math`.